### DISCUSSION TITLE
GetSocial.addOnInitializedListener not working on iOS working on Android

### DISCUSSION THREAD

**Discussion by kobimeridor** [codemagic-ci-cd/codemagic-docs#1557] [Q&A] [answered]
Hello,
using following function to `init` `getSocial` in my App
Working OK on Android
On iOS didn't finish and didn't continue to C
Any Idea?

    initGetsocialPhase = 'B';
    GetSocial.addOnInitializedListener(() async {
    initGetsocialPhase = 'C';

@maks

**Comment by maks** [comment, marked as answer]
@kobimeridor This is more a general Flutter development question versus something specific to building on Codemagic, so it's probably better to ask about that on StackOverflow, though I would suggest that you add more context and details regarding the problem when asking there, to help people assist you with an answer.
